### ISSUE TITLE
loss is not decreasing

### ISSUE THREAD

**Issue by Transtars** [sribdcn/anoymous_MIP_method#2]
Dear Authors,

I hope this message finds you well. Recently, I've had the privilege to work on the implementation of your project. I have been diligently conducting experiments as per the method outlined in your paper. For my setup, I utilized the IP dataset with 240 instances for training and 60 instances for validation. I've set the batch size at 8, maxStoredSol at 500, and a learning rate of 0.003. I've noticed that the model's loss stabilizes around 341 without further decrease. I was wondering if this might be attributed to my choice of hyperparameters or perhaps the size of the training dataset.

I would greatly appreciate your insights or suggestions on this matter. Thank you for your time and consideration.

Warm regards,

Transtars

**Comment by PKUTAN** [comment]
> Dear Authors,
> 
> I hope this message finds you well. Recently, I've had the privilege to work on the implementation of your project. I have been diligently conducting experiments as per the method outlined in your paper. For my setup, I utilized the IP dataset with 240 instances for training and 60 instances for validation. I've set the batch size at 8, maxStoredSol at 500, and a learning rate of 0.003. I've noticed that the model's loss stabilizes around 341 without further decrease. I was wondering if this might be attributed to my choice of hyperparameters or perhaps the size of the training dataset.
> 
> I would greatly appreciate your insights or suggestions on this matter. Thank you for your time and consideration.
> 
> Warm regards,
> 
> Transtars

How do you get the dataset? Thx!

**Comment by Transtars** [comment]
> > Dear Authors,
> > I hope this message finds you well. Recently, I've had the privilege to work on the implementation of your project. I have been diligently conducting experiments as per the method outlined in your paper. For my setup, I utilized the IP dataset with 240 instances for training and 60 instances for validation. I've set the batch size at 8, maxStoredSol at 500, and a learning rate of 0.003. I've noticed that the model's loss stabilizes around 341 without further decrease. I was wondering if this might be attributed to my choice of hyperparameters or perhaps the size of the training dataset.
> > I would greatly appreciate your insights or suggestions on this matter. Thank you for your time and consideration.
> > Warm regards,
> > Transtars
> 
> How do you get the dataset? Thx!

I'm sorry for not being able to reply in a timely manner. I downloaded the dataset from this website: https://drive.google.com/file/d/1MytdY3IwX_aFRWdoc0mMfDN9Xg1EKUuq/view
This link was found from this website:
https://github.com/ds4dm/ml4co-competition/blob/main/DATA.md
